app/views/admin/pages/_form.html.erb

```diff
@@ -38,9 +38,43 @@
     </div>
     <div class="form-group">
       <%= form.label :is_top_level_path %>
-      <%= form.check_box :is_top_level_path %>
-      <p>(Determines if it is accessible by <code>/page-slug</code> vs <code>/page/page-slug</code>) Be careful! ⚠️</p>
+      <%= form.check_box :is_top_level_path, class: "crayons-checkbox" %>
+      <p>Determines if it is accessible by <code>/page-slug</code> vs <code>/page/page-slug</code> Be careful! ⚠️</p>
     </div>
+    <% if @page.errors.count > 0 %>
+      <div id="add-landing-page-link-modal-root">
+        <% @page.errors %>
+      </div>
+    <% end %>
+      <span data-controller="modal" data-modal-root-selector-value="#add-landing-page-link-modal-root" data-modal-content-selector-value="#add-landing-page-link-modal"
+            data-modal-title-value="There's another locked screen..." data-modal-size-value="default">
+        <div class="form-group">
+          <%= form.label :landing_page, "Use as 'Locked Screen'" %>
+          <%= form.check_box :landing_page, class: "crayons-checkbox", data: { action: @page.has_a_landing_page? ? "modal#toggleModal" : "" } %>
+          <p>Determines if this page will be used as a landing page for anonymous viewers.</p>
+        </div>
+        <div class="form-group">
+          <%= form.label :overwrite, "Overwrite" %>
+          <%= form.check_box :overwrite, class: "crayons-checkbox" %>
+        </div>
+        <div id="add-landing-page-link-modal" class="hidden">
+          <div class="form-group grid p-6 mb-6 gap-1">
+            <div class="form-group grid p-6 mb-6 gap-1">
+            <p>
+              Only <b>one</b> page at a time can be used as a "locked screen." If you proceed, this page will no longer show as "locked screen":
+              <br><br>
+              <span style="display:block; border:none; padding:20px; background-color:#F9FAFA;">
+                <%= link_to "Current Locked Screen", @page.landing_page_path %>
+              </span>
+              <br>
+               <button type="button" class="crayons-btn" onclick="toggleOverwrite(); return false;">Overwrite current locked screen</button>&nbsp;
+               <button class="crayons-btn crayons-btn--secondary">Dismiss</button>
```

Comment:
I had to go through adding buttons to my modals as well, because our modals aren't confirmation prompts, they're just modals. Unfortunately I haven't codified into an article yet but I strongly suggest to checkout how we (with Suzanne's help!) managed to implement those in https://github.com/forem/forem/pull/13807 - as it's far from obvious.

Unfortunately these don't work correctly and they are not supposed to, since the modal is just the window and the close button, nothing else. 

A few pointers:

- https://github.com/forem/forem/blob/main/app/views/admin/articles/_pinned_article_modal.html.erb
- https://github.com/forem/forem/blob/main/app/javascript/admin/controllers/article_pinned_modal_controller.js (this inherits from the modal controller and adds behavior)

Let me know if you need further help :)